package.json

```diff
@@ -7,7 +7,7 @@
     "start": "node js-performance.js",
     "test": "mocha spec.setup.js **/*.spec.js",
     "tdd": "mocha --recursive --watch spec.setup.js **/*.spec.js",
-    "lint": "eslint '**/*.js'"
+    "lint": "eslint --fix '**/*.js'"
```

Comment:
Not sure if I like the auto-fix...actually I don't like it. One thing that burned into my mind from our common mentor is that you want to fail on errors until you fixed them. This is analogous to the cron-job-inode-issue. You remember?